### PR TITLE
Revert "#167092464:  Remove userId param from get profile route"

### PR DESCRIPTION
**Pull request by Tyak99**
### What does this PR do?

* Revert changes to the state of develop before `"#167092464:  Remove userId param from get profile route"`

### Description of Task to be completed?

* add userId to the get profile route
* update the test to match the changes 

### Any background context you want to provide?

After making the changes, It cleared that the userId is needed no for a user getting their profile but for user's getting the profile of others. For that to work the userId is needed.

### What are the relevant Github issues?

N/A

### Questions:

N/A

### CODE REVIEW

File: server/routes/api/profiles.js [hunk 1]
```diff
@@ -9,6 +9,6 @@ const { newProfileValidator } = profileValidator;
 const { editProfile, getProfile } = ProfileController;
 
 router.put('/profiles', verifyToken, newProfileValidator, editProfile);
-router.get('/profiles/', verifyToken, getProfile);
+router.get('/profiles/:userId', verifyToken, getProfile);
```
Comment (fortunee) [inline server/routes/api/profiles.js:12]: I thought the purpose of this PR is to get rid of the `userId` query params?

**Review by fortunee — CHANGES_REQUESTED** [review]
Check out my comments

File: server/routes/api/profiles.js (hunk 1 above)
Comment (Tyak99) [reply server/routes/api/profiles.js:12]: Yes. I realized we need the userId after I was told by the team that we need users to be able to view other user's profile

**Review by fortunee — APPROVED** [review]
LGTM

**Pull request merged**